### ISSUE TITLE
FSS-1000 during fine-tuning

### ISSUE THREAD

**Issue by sfengpeng** [niejiahao1998/IFA#5]
We noticed that the samples used for fine-tuning in the FSS-1000 dataset also show up in the test set. Is this a common practice?

**Comment by niejiahao1998** [comment]
We strictly follow the setup proposed in PATNet. The dataloaders for both training and testing are inherited from PATNet.

**Comment by niejiahao1998** [comment]
By the way, during the fine-tuning, we only used one sample in 1-shot setting.

**Comment by sfengpeng** [comment]
However, the code indicates that more than one sample is used. Does the code align with the paper?

**Comment by niejiahao1998** [comment]
Could you please provide the specific position of the code you mentioned?

**Comment by sfengpeng** [comment]
See in deepglobe_ifa.py, the len function returns 6, and in fss_ifa.py the len function returns 520.

**Comment by niejiahao1998** [comment]
Deepgglobe has 6 categories, we load 1 image for each category. Thus the length is 6. We conduct the same process for FSS-100, thus the length is not 1.

**Comment by sfengpeng** [comment]
Thank you for your help. I am new to this field and have one last question. In the FSS-1000 dataset, many samples used for fine-tuning also appear in the testing phase. Is this considered standard practice?

**Comment by niejiahao1998** [comment]
This may caused by the limited images in FSS-1000. Each category of FSS-1000 only contains 10 images. You may add some code to avoid what you observed. But the results will be very similar.

**Comment by sfengpeng** [comment]
I see, thank you

**Comment by sfengpeng** [comment]
> This may caused by the limited images in FSS-1000. Each category of FSS-1000 only contains 10 images. You may add some code to avoid what you observed. But the results will be very similar.

Hi, have you ever conducted a similar test ? I can sense that the fine-tuning approach for the FSS-1000 dataset must have been verified to ensure it doesn't significantly affect the results.
